command/sub8_missions/nodes/tp.yaml

```diff
@@ -0,0 +1,6 @@
+bin: -24 14 -3.5
```

Comment:
Add a comment in this file about what this data represents, what frame, units, and order of coordinates.